normalize_repos/labels.json

```diff
@@ -0,0 +1,163 @@
+{
+  "groups": [
+    {
+      "name": "priority",
+      "labels": [
+        {
+          "name": "critical",
+          "color": "UNFAVOURABLE",
+          "description": "Issue breaks production and needs fixing asap",
+          "emoji": "\uD83D\uDFE5"
+        },
+        {
+          "name": "high",
+          "color": "NEGATIVE",
+          "description": "Issue stalls works on the project or its dependents",
+          "emoji": "\uD83D\uDFE7"
+        },
+        {
+          "name": "medium",
+          "color": "NEUTRAL",
+          "description": "Issue is not up actively harmful but that can escalate",
+          "emoji": "\uD83D\uDFE8"
+        },
+        {
+          "name": "low",
+          "color": "POSITIVE",
+          "description": "Issue doesn't hurt anyone",
+          "emoji": "\uD83D\uDFE9"
+        }
+      ]
+    },
+    {
+      "name": "status",
+      "labels": [
+        {
+          "name": "ready for dev",
+          "color": "LIGHT",
+          "description": "Issue is ready for work as blockers have been resolved",
+          "emoji": "\uD83C\uDFC1"
+        },
+        {
+          "name": "blocked",
+          "color": "MEDIUM",
+          "description": "Issue is not ready for work due to a PR or issue blocking it",
+          "emoji": "\uD83D\uDEA7"
+        },
+        {
+          "name": "ticket work required",
+          "color": "DARK",
+          "description": "Issue needs details and expansion before it can be worked on",
+          "emoji": "\uD83E\uDDF9"
+        },
+        {
+          "name": "discarded",
+          "color": "LIGHTER",
+          "description": "Issue will not be addressed or worked on",
+          "emoji": "⛔️"
+        },
+        {
+          "name": "awaiting triage",
+          "color": "DARKER",
+          "description": "Issue has not yet been ratified and is not ready for work",
+          "emoji": "\uD83D\uDEA6"
+        }
+      ]
+    },
+    {
+      "name": "goal",
+      "color": "ffffff",
+      "labels": [
+        {
+          "name": "addition",
+          "description": "A new feature or component for the application",
+          "emoji": "\uD83C\uDF1F"
+        },
+        {
+          "name": "improvement",
+          "description": "Changes to an existing feature or component in the application",
```

Comment:
"Changes to an existing feature" seems simpler

Reply:
Updated in caf892e.